APMrover2/radio.cpp

```diff
@@ -138,14 +118,14 @@ void Rover::read_radio()
     // check that RC value are valid
     control_failsafe(channel_throttle->get_radio_in());
     // copy RC scaled inputs to outputs
-    SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, channel_throttle->get_control_in());
-    SRV_Channels::set_output_scaled(SRV_Channel::k_steering, channel_steer->get_control_in());
+    g2.motors.set_throttle(channel_throttle->get_control_in());
```

Comment:
We can potentially have a race condition in auto modes, But I don't remember noticing it. Anyway it will be solve with control modes coming next.